data/items/items.xml

```diff
@@ -686,15 +686,17 @@
 	<item id="1497" article="a" name="magic wall">
 		<attribute key="type" value="magicfield" />
 		<attribute key="decayTo" value="0" />
-		<attribute key="duration" value="20" />
```

Comment:
it's nice to have this feature but please don't modify the duration of existing items unless they get official change in rl servers

Reply:
> it's nice to have this feature but please don't modify the duration of existing items unless they get official change in rl servers

Thanks for reviewing <3, I made the changes because it mentions it @EPuncker on the PR 3148

Reply:
Alright, then looks good to me. Sorry, I didn't knew that. Checked on the wiki and nothing was there either.

Reply:
> Sorry, I didn't knew that. Checked on the wiki and nothing was there either.

https://github.com/otland/forgottenserver/issues/3148#issuecomment-667573270

Reply:
perfect, I'll fix it later, thanks <3

Reply:
> Sorry, I didn't knew that. Checked on the wiki and nothing was there either.

I would like to make the changes, but the repository i deleted it by accident, I suppose I will have to redo everything again, or I don't know what to do about it. thanks!

Reply:
@MillhioreBT If you are recovering a personal repository that is not part of an organization, start by opening “Settings” in the top right dropdown menu. Click on the “Repositories” menu item. From that screen, you should see a section for deleted repositories in the top bar, click on that.

Reply:
@EPuncker I will try, although I have doubts to achieve it, it can be said that I am the baddest woman using git and github 😭